packages/charts/src/chart_types/xy_chart/state/selectors/get_cursor_pointer.ts

```diff
@@ -48,11 +51,11 @@ export const getPointerCursorSelector = createCustomCachedSelector(
       return 'pointer';
     }
 
-    // no pointer if there is a onAnnotionClick but there is a highlighted geometry
+    // no pointer if there is a onAnnotationClick but there is a highlighted geometry
     if (highlightedGeometries.length > 0 && (!settingsSpec.onElementClick || !settingsSpec.onElementOver)) {
       return DEFAULT_CSS_CURSOR;
     }
-    if (settingsSpec.onAnnotationClick) {
+    if (settingsSpec.onAnnotationClick && tooltipState) {
       return 'pointer';
     }
```

Comment:
I see why this is here now. I think you can simplify this logic to just this...

```suggestion
    if (highlightedGeometries.length === 0 && settingsSpec.onAnnotationClick && tooltipState) {
      return 'pointer';
    }
```

Are you sure you don't need to check the `tooltipState` for anything? Just if it exists that means it has something?

Reply:
Yep! Good call cb44d3e